### PR TITLE
feat: remove "ask api" command and move export-package and upload-cat…

### PR DESCRIPTION
**Pull request by RonWang**

### CODE REVIEW

File: docs/concepts/Smapi-Command.md [hunk 1]
```diff
@@ -101,44 +119,160 @@ The path params
...
+### list-catalogs-for-skill
 
-Create a new upload for a catalog and returns location to track the upload process.
+Lists all the catalogs associated with a skill.
 
-`create-catalog-upload` command format:
+`list-catalogs-for-skill` command format:
 
-`$ askx smapi create-catalog-upload <-c|--catalog-id <catalog-id>> <--catalog-upload-request-body <catalog-upload-request-body>> [-p| --profile <profile>] [--debug]`
+`$ askx smapi list-catalogs-for-skill [--next-token <next-token>] [--max-results <max-results>] <-s|--skill-id <skill-id>> [-p| --profile <profile>] [--debug]`
```
Comment (pbheemag) [inline docs/concepts/Smapi-Command.md:200]: shouldn't this be just `ask`? change wherever applicable.

File: docs/concepts/Smapi-Command.md [hunk 2]
```diff
@@ -166,6 +300,282 @@ Gets detailed information about an upload which was created for a specific catal
...
+
+`$ askx smapi generate-catalog-upload-url <-c|--catalog-id <catalog-id>> <--number-of-upload-parts <number-of-upload-parts>> [-p| --profile <profile>] [--debug]`
+
+**Options**
+
+<dl>
+    <dt>-c,--catalog-id <catalog-id></dt>
+    <dd markdown="span">[REQUIRED] Unique identifier of the catalog.</dd>
+    <dt>--number-of-upload-parts <number-of-upload-parts></dt>
+    <dd markdown="span">[REQUIRED] The number of parts the file will be split into. An equal number of presigned upload urls will be generated in response to facilitate each part's upload.</dd>
```
Comment (pbheemag) [inline docs/concepts/Smapi-Command.md:550]: nit: `pre-signed`. Change wherever applicable.

File: bin/ask.js [hunk 3]
```diff
@@ -18,13 +18,12 @@ require('@src/commands/dialog').createCommand(commander);
...
-    .command('api', 'list of Alexa Skill Management API commands')
     .command('smapi', 'list of Alexa Skill Management API commands')
     .command('util', 'tooling functions when using ask-cli to manage Alexa Skill')
     .version(require('../package.json').version)
     .parse(process.argv);
 
-const ALLOWED_ASK_ARGV_2 = ['api', 'configure', 'deploy', 'new', 'init', 'dialog', 'smapi', 'util', 'help', '-v', '--version', '-h', '--help'];
+const ALLOWED_ASK_ARGV_2 = ['configure', 'deploy', 'new', 'init', 'dialog', 'smapi', 'util', 'help', '-v', '--version', '-h', '--help'];
 if (process.argv[2] && ALLOWED_ASK_ARGV_2.indexOf(process.argv[2]) === -1) {
     console.log('Command not recognized. Please run "askx" to check the user instructions.');
```
Comment (pbheemag) [inline bin/ask.js:28]: guess i missed this in my previous PR. can you change this to `ask`

File: docs/concepts/Smapi-Command.md (hunk 2 above)
Comment (RonWang) [reply docs/concepts/Smapi-Command.md:550]: this is defined from the smapi-sdk .. CLI cannot change it..

File: bin/ask.js (hunk 3 above)
Comment (RonWang) [reply bin/ask.js:28]: sure

File: docs/concepts/Smapi-Command.md (hunk 1 above)
Comment (RonWang) [reply docs/concepts/Smapi-Command.md:200]: good call, changed in the new commit

**Pull request merged**